simulateCoOp.py

```diff
@@ -0,0 +1,176 @@
+import time
+
+startTime = time.time() 
+
+import pandas as pd
+import numpy as np
+import statistics as stats
+import matplotlib.pyplot as plt
+import importlib
+
+import farm as farm
+import farmRe as farmRe
+import functions as functions
+
+#importlib.reload(farm)
+
+def compileCoOp(farmStr): # strategyStr = None, treeStr = None):
```

Comment:
hm. this is like a data-loader function loads in a csv and returns a list of farms (why not a dataframe, now that we're thinking of it as a loader?), should be named that and moved outside this file.

Reply:
at first I was like... oh maybe we're avoiding dataframes altogether? but we're not. So, if this was inside `functions.py`, you've already got the right import up top ;D